types/parse-json/index.d.ts

```diff
@@ -1,9 +1,11 @@
 // Type definitions for parse-json 4.0
 // Project: https://github.com/sindresorhus/parse-json
 // Definitions by: mrmlnc <https://github.com/mrmlnc>
+//                 enderlee <https://github.com/chnliquan>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare function parseJson(input: string | null, filepath?: string): any;
 declare function parseJson(input: string | null, reviver: (key: any, value: any) => any, filepath?: string): any;
+declare namespace parseJson {}
```

Comment:
I would not recommend this. The `import * as` syntax in ES modules is intended to import a "module namespace object", which would *not* be callable if using actual ES modules. Since this is a CommonJS-style export (using the TypeScript `export =` syntax, I would recommend you use the TypeScript `import=` syntax instead in your own code:

```ts
// INCORRECT (ES Module syntax):
// import * as parseJson from "parse-json"; 

// CORRECT (TS CommonJS syntax):
import parseJson = require("parse-json");

parseJson(...);
```